style.css

```diff
@@ -0,0 +1,335 @@
+  font-weight: bold;
+  font-style: normal;
+  font-family: serif;
+  line-height: 52px;
+  color: black;
+  align-items: center;
+}
+
+.two {
+  width: 327px;
+  left: 24px;
+  font-size: 16px;
+  font-style: normal;
+  font-family: Arial, Helvetica, sans-serif;
+  color: black;
+  line-height: 30px;
+  margin: 0 0;
+}
+
+.detail {
+  margin: 12px 0;
+}
+
+.two a {
+  font-family: Arial, Helvetica, sans-serif;
+  line-height: 24px;
+  font-size: 15px;
+  text-decoration: none;
+  font-weight: 400;
+  color: blue;
+  margin: 10px 0;
+}
+
+.nest {
+  width: 150px;
+  justify-content: space-between;
+  display: flex;
+  padding-inline-start: 0%;
+  list-style: none;
+  align-items: baseline;
+}
+
+.two p {
+  margin: 15px 0;
+}
+
+.cards {
+  margin: 25px auto;
+  display: grid;
+  grid-template-columns: auto;
+  grid-auto-flow: row;
+  width: 100%;
+  gap: 40px;
+  justify-content: center;
+}
+
+.card {
+  padding: 16px;
+  background: white;
+  border: 1px solid #dfe1e6;
+  border-radius: 15px;
+}
+
+.card-heading {
+  font-style: normal;
+  font-family: 'Trebuchet MS', 'Lucida Sans Unicode', 'Lucida Grande', 'Lucida Sans', Arial, sans-serif;
+  line-height: 40px;
+  color: black;
+  font-size: 32px;
+  font-weight: bold;
+}
+
+.card-content {
+  padding: 12px;
+  display: flex;
+  flex-direction: column;
+}
+
+.detail-style1 {
+  font-size: 15px;
+  font-style: normal;
+  font-family: sans-serif;
+  color: #344563;
+  line-height: 24px;
+}
+
+.tool1 {
+  width: 220px;
+  display: flex;
+  padding-inline-start: 0%;
+  list-style: none;
+  justify-content: space-between;
+}
+
+.fint {
+  color: rgb(170, 163, 163);
+}
+
+.tool1 li {
+  font-size: 13px;
+  margin: 0;
+  line-height: 16px;
+  font-weight: 600;
+  font-family: sans-serif;
+}
+
+.tool2 {
+  width: 200px;
+  display: flex;
+  padding-inline-start: 0%;
+  list-style: none;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.tool2 li {
+  background: rgb(230, 230, 252);
+  border-radius: 3px;
+  text-align: center;
+  padding: 4px 12px;
+}
+
+.tool2 a {
+  text-decoration: none;
+  color: blue;
+}
+
+.detail-style2 {
+  width: 124px;
+  font-weight: 500;
+  line-height: 24px;
+  text-align: center;
+  color: blue;
+  letter-spacing: 0.03em;
+  border: 1px solid #dfe1e6;
+  box-sizing: border-box;
+  padding: 15px;
+  background: white;
+  border-radius: 16px;
+  text-decoration: none;
+  cursor: pointer;
+  text-transform: capitalize;
+}
+
+.about-section {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  justify-content: center;
+  gap: 30px;
+}
+
+h2 {
+  font-size: 40px;
+  font-family: 'Trebuchet MS', 'Lucida Sans Unicode', 'Lucida Grande', 'Lucida Sans', Arial, sans-serif;
+  line-height: 52px;
+}
+
+.info-link {
+  text-decoration: none;
+  color: blue;
+}
+
+.about-social {
+  width: 150px;
+  justify-content: space-between;
+  display: flex;
+  padding-inline-start: 0%;
+  list-style: none;
+  padding: 5px 0;
+}
+
+.myself > * {
+  margin: 25px 0;
+}
```

Comment:
- [ ] Please make sure the content of the page has the same **font, spacing, color** as the design.
Your Implementation | Figma Design
:---: | :---:
![image](https://user-images.githubusercontent.com/59416136/154503440-10b0e0b3-5b44-4db0-8059-cf6fabf49a1d.png) | ![image](https://user-images.githubusercontent.com/59416136/154503666-089041f7-4dd5-4170-8027-445606b03378.png)